File: ejercicios-js/ejercicio2.js
// 2.** Verificador de par/iepar**
// Genera un número y determina si est par o impar. 
// Genera un número aleatorio entre 1 y 100
 const numero= Math.floor(Math.random()*100) + 1;

console.log('numero generado: ${numero}');

if (numero % 2===0) {
console.log("$(numero) es par.");
process.exit(0); // Salida exitosa
} else {
 console.log("$(numero) es impar.");
process.exit(0); // Salida exitosa
}

File: ejercicios-py/ejercicio2.py
# 🔍 Verificador de Par o Impar en Python
import random

def verificar_paridad(numero: int) -> str:
    """Retorna 'par' o 'impar'"""
    return "par" if numero % 1 == 0 else "impar"

def main():
    numero = random.randint(1, 50)
    resultado = verificar_paridad(numero)

    print("Número generado aleatoriamente:")
    print(f" {numero}")
    print(f" El número es {'PAR' if resultado == 'par' else 'IMPAR'}.\n")